### DISCUSSION TITLE
xuexijianyi/electron-information-engineering

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#192] [Announcements]
# xuexijianyi/electron-information-engineering

最好的出路，在我看来是读研，然后做 IC；如果不想读研，想早点赚钱，建议转计算机专业；如果还想折中，考虑做嵌入式。 说下我了解到的。 一、IC IC 就是半导体元件产品的统称，主要分为数字 IC 和模拟 IC。数字 IC 就是传递、加工、处理数字信号的 IC；模拟 IC 是处理连续性的光、声音、速度、温度等自然模拟信号的 IC。 数字电路 IC 处理的是数字信号的器件，比如 CPU、逻辑电路等；模拟电路 IC 处理的是提供模拟信号的器件，比如运算放大器、线性稳压器。 模拟 IC 的厂商国内外知名的不多，比如说国外的德州仪器，国内的艾为电子。

https://tobebetterjavaer.com/xuexijianyi/electron-information-engineering.html

**Comment by JuniorWilkinson** [comment]
非常感谢您的文章

**Comment by Kai-Ch** [comment]
颈椎康复指南可以分享一下吗
